js/on-off.js

```diff
@@ -1,6 +1,8 @@
 $(document).ready(function(){
- 	var check;
+   var check;
+   var check2;
```

Comment:
This is a very confusing name for variable.

Reply:
Done, variable renamed to `statuscheck`